panda-whisker/Trestlebridge-Farms
# Ds ticket14

# Description

Fixed sunflower purchase issue. You can put it in both fields now. Added some "styling" and ascii images

Fixes # (ticket 14)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

To see all of the ASCII images, throw the errors for not selecting a facility, an animal and a seed.

To test adding the sunflower:

FIRST
1. Create Facility -> 2. Plowed Field
THEN
1. Create Facility -> 3. Natural Field
THEN
3. Purchase Seeds -> 3. Sunflower -> 1. Plowed Field
THEN
3. Purchase Seeds -> 3. Sunflower -> 1. Natural Field
THEN
4. Display Farm Status 

# Checklist:
